AzureCalling/Controllers/AudioDeviceSelectionViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class AudioDeviceSelectionViewController: UIViewController, UITableViewDelegate {
+
+    // MARK: Properties
+
+    var audioDeviceSelectionManager: AudioDeviceSelectionManager!
+    var audioDeviceTableDataSource: TableViewDataSource!
```

Comment:
Let's move audioDeviceTableDataSource to this class rather than using an object hold it